Toolset/libraries/revcommonlibrary.livecodescript

```diff
@@ -224,6 +214,7 @@ setProp cREVGeneral[pWhichProp] pWhichProfile
       unlock messages
       exit cREVGeneral
    end if
+   
    if pWhichProp is "breakPoints" then
       
       pass cREVGeneral
```

Comment:
Since we are doing cleanup, should we remove this whole section of code (`breakPoints`)?  The pass could be handled at the top with the initial check for the properties that are handled. 230/221 to 271/262 could be removed as the code is never reached at a minimum.

Reply:
Yes you are right this can all be removed and just remove breakpoints from the condition at the top. This should have been cleaned up when the pass was put in.